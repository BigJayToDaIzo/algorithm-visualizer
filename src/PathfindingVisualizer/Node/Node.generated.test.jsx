import renderer from 'react-test-renderer';
import React, { Component } from 'react';

import Node from './Node';

jest.mock('./Node.css');

const renderTree = tree => renderer.create(tree);
describe('<Node>', () => {
  it('should render component', () => {
    expect(renderTree(<Node 
    />).toJSON()).toMatchSnapshot();
  });
  
});