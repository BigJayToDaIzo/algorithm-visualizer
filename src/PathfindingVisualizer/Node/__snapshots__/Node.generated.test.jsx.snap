// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Node> should render component 1`] = `
<div
  className="node "
  id="node-undefined-undefined"
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseUp={[Function]}
/>
`;
